### ISSUE TITLE
errors when handling missing meta/main.yml 

### ISSUE THREAD

**Issue by rl201** [nickjj/ansigenome#54]
roles without meta/main.yml files seem to produce spurious dependencies. I think this is due to a mistake in scan.py around line 202:

**Comment by rl201** [comment]
if not os.path.exists(self.paths["meta"]):
    return ""
should perhaps be
if not os.path.exists(self.paths["meta"]):
    self.dependencies=[]
    return ""

**Comment by nickjj** [comment]
Are you saying it's reporting dependencies for the wrong role as is?

**Comment by rl201** [comment]
> Are you saying it's reporting dependencies for the wrong role as is?

Yes. If role A depends, via meta/main.yml, on role B but role B has no 
meta/main.yml file, role B is reported as depending on itself. Creating an 
empty meta/main.yml yields the expected behaviour.

**Comment by nickjj** [comment]
I'd happily accept a PR if you wanted create one.
